channel.py

```diff
@@ -554,9 +558,28 @@ def export_tier_prices_to_magento(self):
         Exports tier prices of products from tryton to magento for this channel
         :return: List of products
         """
+        ChannelListing = Pool().get('product.product.channel_listing')
+
         self.validate_magento_channel()
 
-        for listing in self.product_listings:
+        price_domain = [
+            ('channel', '=', self.id),
+        ]
+
+        if self.magento_last_tier_price_export_time:
+            price_domain.append(
+                (
+                    'product.write_date', '>=',
```

Comment:
cost_price and list_price are in templates so wont change write date for product. but it will work. Also It will be changed with the patch for adding these two fields in product variant.